src/SignalR/clients/ts/signalr/src/HubConnectionBuilder.ts

```diff
@@ -44,15 +69,18 @@ export class HubConnectionBuilder {
 
     /** Configures custom logging for the {@link @aspnet/signalr.HubConnection}.
      *
-     * @param {LogLevel | ILogger} logging An object implementing the {@link @aspnet/signalr.ILogger} interface or {@link @aspnet/signalr.LogLevel}.
+     * @param {LogLevel | string | ILogger} logging A {@link @aspnet/signalr.LogLevel}, a string representing a LogLevel, or an object implementing the {@link @aspnet/signalr.ILogger} interface. See {@link https://docs.microsoft.com/en-us/aspnet/core/signalr/configuration#configure-logging|the documentation for client logging configuration} for more details.
      * @returns The {@link @aspnet/signalr.HubConnectionBuilder} instance, for chaining.
      */
-    public configureLogging(logging: LogLevel | ILogger): HubConnectionBuilder;
-    public configureLogging(logging: LogLevel | ILogger): HubConnectionBuilder {
+    public configureLogging(logging: LogLevel | keyof typeof LogLevelNameMapping | ILogger): HubConnectionBuilder;
```

Comment:
I still wonder if we need to export LogLevelNameMapping if we keep the `keyof typeof LogLevelNameMapping` in one of the public signatures. I'd just remove it.

Reply:
This is just a derp. The fact that this signature even exists is an artifact of a previous change.

```suggestion
```